packages/plugin-vue-jsx/package.json

```diff
@@ -28,12 +28,15 @@
   },
   "dependencies": {
     "@rsbuild/plugin-babel": "workspace:*",
-    "@vue/babel-plugin-jsx": "1.2.1"
+    "@vue/babel-plugin-jsx": "1.2.1",
+    "babel-plugin-vue-jsx-hmr": "1.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.23.2",
     "@rsbuild/core": "workspace:*",
+    "@rspack/core": "0.5.5",
     "@scripts/test-helper": "workspace:*",
+    "@types/babel__core": "^7.20.3",
```

Comment:
can we delete this

Reply:
sure, if replaced npm package, it is useless

Reply:
already removed [df85101](https://github.com/web-infra-dev/rsbuild/pull/2018/commits/df85101efcb8253cb2afd74965922e2c3c7206b1)